### PR TITLE
docs for named login MFA methods improvement

### PR DESCRIPTION
**Pull request by hghaf099**
Addresses https://hashicorp.atlassian.net/browse/VAULT-11846

### CODE REVIEW

**Comment by hghaf099** [comment]
> Member

For now, it is meant for 1.13 release. If we decided to backport the feature, we are going to add labels to it later on.

File: website/content/docs/auth/login-mfa/index.mdx [hunk 1]
```diff
@@ -62,8 +62,10 @@ In the Single-phase login, the required MFA information is embedded in a login r
 the `X-Vault-MFA` header. In this case, the MFA validation is done
 as a part of the login request.
 
-MFA credentials are retrieved from the `X-Vault-MFA` HTTP header. The format of
-the header is `mfa_method_id[:passcode]`. The item in the `[]` is optional. If there are multiple MFA methods that need to be validated, a user can pass in multiple `X-Vault-MFA` HTTP headers.
+MFA credentials are retrieved from the `X-Vault-MFA` HTTP header. Before Vault 1.13.0, the format of
+the header is `mfa_method_id[:passcode]` for TOTP, Okta, and PingID. However, for Duo, it is `mfa_method_id[:passcode=<passcode>]`.
+The item in the `[]` is optional. From Vault 1.13.0, the format is consistent for all supported MFA methods, and one can use either of the above two formats.
+If there are multiple MFA methods that need to be validated, a user can pass in multiple `X-Vault-MFA` HTTP headers.
```
Comment (tjperry07) [inline website/content/docs/auth/login-mfa/index.mdx:68]: MFA credentials are retrieved from the `X-Vault-MFA` HTTP header. Before Vault 1.13.0, the format of
the header is `mfa_method_id[:passcode]` for TOTP, Okta, and PingID. However, for Duo, it is `mfa_method_id[:passcode=<passcode>]`.

This reads as before vault 1.13.0 the duo method is different. Is that correct?

**Review by tjperry07 — CHANGES_REQUESTED** [review]
Since the change starts in 1.13.x, make it clear this is 1.13.x behavior and this is 1.12.x behavior.

File: website/content/docs/auth/login-mfa/index.mdx (hunk 1 above)
Comment (hghaf099) [reply website/content/docs/auth/login-mfa/index.mdx:68]: Yes, that is correct

**Review by tjperry07 — APPROVED** [review]

**Comment by hghaf099** [comment]
I am closing this ticket in favour of a new doc ticket.

**Pull request closed without merging**